webapp/components/wpt-amend-metadata.js

```diff
@@ -256,19 +256,34 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
     return link;
   }
 
-  hasHref(product) {
-    return product === 'chrome';
+  hasSearchURL(product) {
+    return product === 'chrome' || product === 'edge' || product === 'firefox' ||
```

Comment:
Looks like we will support search for all browsers?

Reply:
If that is the case, perhaps we can remove the `if="[[hasSearchURL(node.product)]]"`